app/models/article.rb

```diff
@@ -2,31 +2,41 @@
 #
 # Table name: articles
 #
-#  id                       :bigint           not null, primary key
-#  content                  :text
-#  description              :text
-#  status                   :integer
-#  title                    :string
-#  views                    :integer
-#  created_at               :datetime         not null
-#  updated_at               :datetime         not null
-#  account_id               :integer          not null
-#  author_id                :bigint
-#  category_id              :integer
-#  folder_id                :integer
-#  portal_id                :integer          not null
+#  id                    :bigint           not null, primary key
+#  content               :text
+#  description           :text
+#  status                :integer
+#  title                 :string
+#  views                 :integer
+#  created_at            :datetime         not null
+#  updated_at            :datetime         not null
+#  account_id            :integer          not null
+#  associated_article_id :bigint
```

Comment:
let's use the same naming conventions as we have used in categories. so `linked_article_id` ? 
or else the synonyms could start causing confusion